src/types/PasteConverter.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+#include "inc/PasteConverter.h"
+
+using namespace Microsoft::Console::Utils;

```

Comment:
I don't honestly know where to put this class. Right now it is heavily VT-related. But in the future maybe other features will be added, making it less VT-related. So perhaps `Types` is also OK?

Reply:
I think we should keep this VT-specific for now. It can be refactored later if need be.